api/server/v1/server.proto

```diff
@@ -181,12 +181,42 @@ message Settings {
   uint32 default_role_id = 18;
 }
 
+// Settings represents a stripped-down version of PMM Server settings that can be accessed by users of all roles.
+message ReadOnlySettings {
+  // True if updates are enabled.
+  bool updates_enabled = 1;
+  // True if telemetry is enabled.
+  bool telemetry_enabled = 2;
+  MetricsResolutions metrics_resolutions = 3;
+  google.protobuf.Duration data_retention = 4;
+  // True if Advisor is enabled.
+  bool advisor_enabled = 7;
+  // Percona Platform user's email, if this PMM instance is linked to the Platform.
+  string platform_email = 8;
+  // True if Alerting is enabled.
+  bool alerting_enabled = 10;
+  // PMM Server public address.
+  string pmm_public_address = 11;
+  // Intervals between Advisor runs.
+  AdvisorRunIntervals advisor_run_intervals = 12;
+  // True if Backup Management is enabled.
+  bool backup_management_enabled = 13;
+  // True if Azure Discover is enabled.
+  bool azurediscover_enabled = 14;
+  // True if Access Control is enabled.
+  bool enable_access_control = 17;
```

Comment:
oh shit, it's inconsistent with other fields.

Reply:
Yeah I saw too - will need to prep separate PR to update it